src/ssl.c

```diff
@@ -20217,6 +20217,124 @@ void wolfSSL_GENERAL_NAMES_free(WOLFSSL_GENERAL_NAMES *gens)
     wolfSSL_sk_free(gens);
 }
 
+#ifdef OPENSSL_ALL
+int wolfSSL_GENERAL_NAME_print(WOLFSSL_BIO* out, WOLFSSL_GENERAL_NAME* gen)
```

Comment:
Please add a function comment describing behavior, arguments, and return values, thanks

Reply:
Added comments.